clang/test/CXX/drs/cwg2747.cpp

```diff
@@ -0,0 +1,11 @@
+// RUN: %clang_cc1 -std=c++11 -pedantic-errors -verify=expected %s -E | FileCheck %s --strict-whitespace --allow-empty
```

Comment:
We run DR tests in complete range of C++ versions.

Reply:
You forgot C++14.

Reply:
Fixed now, thanks